ldm/modules/attention.py

```diff
@@ -1,9 +1,10 @@
-from inspect import isfunction
```

Comment:
please don't include style changes such as import rearrangements or whitespace changes in a commit request

Reply:
FYI, you can make git fix that automagically. See -> https://github.com/lstein/stable-diffusion/blob/main/.gitattributes

Reply:
I rolled this back already